packages/backend/src/utils/npm.ts

```diff
@@ -40,11 +40,19 @@ class Command {
   private isInBAS: boolean;
 
   constructor() {
-    this.globalNodeModulesPathPromise = this.execCommand(`${NPM} root -g`);
+    this.setGlobalNodeModulesPath();
     this.SET_DEFAULT_LOCATION = messages.set_default_location(customLocation.DEFAULT_LOCATION);
     this.isInBAS = !_.isEmpty(_.get(process, "env.WS_BASE_URL"));
   }
 
+  private setGlobalNodeModulesPath() {
+    void this.execCommand(`${NPM} root -g`).then((globalNodeModulesPath: string) => {
+      this.globalNodeModulesPathPromise = fs.promises
+        .mkdir(globalNodeModulesPath, { recursive: true })
+        .then(() => globalNodeModulesPath);
+    });
```

Comment:
what is an advantage of using that calling method vs await mkdir(...) ?

Reply:
explained